components/log-viewer-webui/server/src/app.test.ts

```diff
@@ -1,27 +1,27 @@
-import httpStatusCodes from "http-status-codes";
-import {test} from "tap";
+import {StatusCodes} from "http-status-codes";
+import tap from "tap";
 
 import app from "./app.js";
 
 
-test("Tests the example routes", async (t) => {
-    const server = await app({});
+await tap.test("Tests the example routes", async (t) => {
```

Comment:
This change is needed because ESLint does not know `test()` is defined in the global scope in the context of `tap` runs.